client/js/_author/components/assessments/question_types/_question.jsx

```diff
@@ -99,6 +99,8 @@ export default class Question extends React.Component {
     const { name, type, id, question, bankId } = item;
     const className = this.getClassName();
 
+    const questionText = question ? question.text : '';
```

Comment:
This accounts for the question object being potentially null. Which the base deserializer defaults to.